README.md

```diff
@@ -1,3 +1,76 @@
 # Rancher Tests
 
-Welcome to the rancher test repo. 
\ No newline at end of file
+Welcome to the rancher test repo. 
+
+## Branching Strategy
+
+main - active development branch
+
+stable - rebased from main after each rancher/rancher release. This branch should be used when importing this repo
```

Comment:
What if we used a trunk-based strategy? I don't know what having two branches would solve since these are tests we should know right away if something is broken before merging into the main branch.

Reply:
briefly discussed offline, may reconsider if this is a problem in the future.